src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -66,6 +66,12 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_2022-08-17T10:56:45Z.xml
       relativeToChangelogFile: true
+  - include:
+      file: changesets/changelog_2022-11-28T16:04:50Z.xml
+      relativeToChangelogFile: true
+  - include:
```

Comment:
why not ? im doing separate changes

Reply:
Hmm for me it's one changeset by commit, and the PR will be squashed into one commit, but I don't know liquidbase and how works changesets so I can't tell if it will introduce lowest performance or not